### PR TITLE
Convert files in `lms/static/scripts/{browser_check, postmessage_json_rpc}` to TS

### PR DESCRIPTION
**Pull request by robertknight**

### CODE REVIEW

File: lms/static/scripts/postmessage_json_rpc/server.ts [hunk 1]
```diff
@@ -152,10 +153,8 @@ export class Server {
 
   /**
    * Return true if the provided object is a JSON-RPC request.
-   *
-   * @param {any} request
    */
-  _isJSONRPCRequest(request) {
+  _isJSONRPCRequest(request: any) {
```
Comment (acelaya) [inline lms/static/scripts/postmessage_json_rpc/server.ts:157]: Should this method be `private`?

```suggestion
  private _isJSONRPCRequest(request: any) {
```

File: lms/static/scripts/postmessage_json_rpc/server.ts [hunk 2]
```diff
@@ -171,11 +170,8 @@ export class Server {
 
   /**
    * Return a JSON-RPC response object for the given JSON-RPC request object.
-   *
-   * @param {JsonRpcRequest} request
-   * @return {Promise<JsonRpcResponse>}
    */
-  async _jsonRPCResponse(request) {
+  async _jsonRPCResponse(request: JSONRPCRequest): Promise<JSONRPCResponse> {
```
Comment (acelaya) [inline lms/static/scripts/postmessage_json_rpc/server.ts:174]: Should this method be `private`?

```suggestion
  private async _jsonRPCResponse(request: JSONRPCRequest): Promise<JSONRPCResponse> {
```

**Review by acelaya — APPROVED** [review]
Looks good. Just added a couple nitpick comments regarding private method visibility.

File: lms/static/scripts/postmessage_json_rpc/server.ts (hunk 1 above)
Comment (robertknight) [reply lms/static/scripts/postmessage_json_rpc/server.ts:157]: Yes, indeed it should be. Thanks for the reminder!

**Pull request merged**